lighthouse-core/gather/gather-runner.js

```diff
@@ -394,12 +394,11 @@ class GatherRunner {
     try {
       await driver.connect();
       const baseArtifacts = await GatherRunner.getBaseArtifacts(options);
-      await GatherRunner.loadBlank(driver);
-      baseArtifacts.BenchmarkIndex = await options.driver.getBenchmarkIndex();
       await GatherRunner.setupDriver(driver, options);
+      baseArtifacts.BenchmarkIndex = await options.driver.getBenchmarkIndex();
```

Comment:
wait I take it back this won't work :)

It needs to go before we setup emulation since its purpose is establishing the right multiplier (see #6162)

it doesn't seem like it'd be the end of the world to have the two `loadBlank` calls in this function, was there another reason I'm forgetting that it's inside `setupDriver` now?

Reply:
+1 to restoring the original `loadBlank` line

Reply:
done